src/dateutils.js

```diff
@@ -19,7 +19,7 @@ function sameDate(a, b) {
 
 function sameWeek(d1, d2, firstDayOfWeek) {
   const weekDates = getWeekDates(d1, firstDayOfWeek, 'yyyy-MM-dd');
-  return weekDates?.includes(d2);
+  return weekDates && weekDates.includes(d2);
```

Comment:
What is the problem with optional chaning?

Reply:
was part of checking something, brought it back